CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
   * `-store-gateway.sharding-ring.heartbeat-period`
 * [ENHANCEMENT] Memberlist: optimized receive path for processing ring state updates, to help reduce CPU utilization in large clusters. #4345
 * [ENHANCEMENT] Memberlist: expose configuration of memberlist packet compression via `-memberlist.compression=enabled`. #4346
+* [ENHANCEMENT] Hide configuration secret value #4371
```

Comment:
Try to write this so a typical user will understand roughly what changed, and whether it is important to them.
For example, which secret?  Hide it from what, or where?